.github/pull_request_template.md

```diff
@@ -1,23 +1,24 @@
+<!-- Enter a brief description/summary of your PR here. What does it fix/what does it change/how was it tested (even manually, if necessary)? -->
 ## Summary of the Pull Request
 
-**What is this about:**
+<!-- Please review the items on the PR checklist before submitting-->
+## PR Checklist
 
-**What is included in the PR:** 
-
-**How does someone test / validate:** 
-
-## Quality Checklist
-
-- [ ] **Linked issue:** #xxx
-- [ ] **Communication:** I've discussed this with core contributors in the issue. 
+- [ ] **Closes:** #xxx
```

Comment:
Why change this?

Reply:
(speaking for myself) more automation means less manual work?

Reply:
> (speaking for myself) more automation means less manual work?

I don't think this would work when the word is bold

Reply:
I hope that it doesn't work for automation. Otherwise we have to reopen all the issues after merging. Don't think we want this.

Reply:
If that happens we can change the verbiage

Reply:
Not sure if the keyword is "close" or "fix". 🤔

Reply:
@crutkas 
FYI: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

Reply:
> I don't think this would work when the word is bold

You are correct.